x-pack/plugins/session_view/public/components/DetailPanelDescriptionList/styles.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { useMemo } from 'react';
+import { CSSObject } from '@emotion/react';
+
+export const useStyles = () => {
```

Comment:
I won't comment on all the `useStyles`, but just want to note to check all of them after you get @cchaos 's feedback